docs/services/vision/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Use Computer Vision with the Vision Service"
+title: "Computer Vision Service"
```

Comment:
I realize that the service is just called "vision," however was trying to find a new sort of middle ground that takes out the ambiguous wording and some of the consistency while maintaining quick comprehensibility--the link title is already computer vision for this reason.

We refer to the data management service in all sorts of ways throughout the docs such as "data service," "the data manager," "data management service," and "data management," and the config uses the string `data_manager`. I figured this is fine and contextual flexibility is actually a benefit here, that supports readability and concision. I figure the same can be applied here.

I think "with" also creates confusion-- "Computer Vision with the Vision Service" can also be parsed as "Computer Vision Along with the Vision Service" so if you think "computer vision service" is not okay perhaps "Vision Service" is the way to go. However, that defeats the original reason for having "computer" in this title so idk anymore :D

```suggestion
title: "Vision Service"
```

Reply:
On a positive note, since this page is becoming more for reference, it doesn't need such an active title as "Use..." anymore because the related how-to/quickstart content will cover that. So that makes me less concerned about just going with "Vision Service" here on this reference page.

Reply:
Merging but happy to change again if you have other thoughts!